### PR TITLE
Reworks BoxStation medbay and service area

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Reworks some outdated Box room shenanigans:
- Service area rework
- Medbay rework 
## Why It's Good For The Game

BoxStation is old and clunky

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

</details>

## Changelog
:cl: Varo, Bombyy, PigeonVerde
tweak: BoxStation: Reworked the service area and maintenance
tweak: BoxStation: Reworked the medbay area and maintenance
add: BoxStation: a room for the psychiatrist in medbay
tweak: BoxStation: fixed chaplain backroom having a weird shape
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
I don't like the changes to the hallway, since its just a perfectly straight hallway with no inlets, which tends to look amateurly mapped.

**Comment by MarkusLarsson421** [comment]
What is up with Chemistry? I'm sorry, but it looks horrendous.

**Comment by MarkusLarsson421** [comment]
The Botany backroom is a good change. Always such a tight room if there was more than one person on the server. lol

**Review by spockye — CHANGES_REQUESTED** [review]
image

misplaced cabling

image

misplaced cabling

image

atmos fuckery

image

camera on windoor (windoor is also missnamed and ID'd for some reason?)

image

light on window

image

why have two windoors here?

image

if your going to have only flower glass walls, try to vary them a little

image

no pete?

image

having both the mourge and surgery theathre be connected to the all wooden breakroom feels a little off. mabye try moving the breakroom?


-------

in general, not too bad!  mabye try to use a few more different decals? it might be a good idea to re-organize the rooms and re-evaluate the amount of flower walls you use. a couple is always nice, but too many and it can turn into visual clutter

**Review by spockye — COMMENTED** [review]
image

area fucked

image

you gotta decide if you want your maintenance doors to be part of their rooms or maintenance!!!

image
piping error

image
you forgot to move this

image

windows in walls

image

window in wall

image
wood borders spotted in the wild

image

you can rotate these decals!

image

missing decal

**Review by spockye — CHANGES_REQUESTED** [review]
image
missing firelocks

image

wall in table

image

this hallway could use a airalarm

image

image

these rooms need fire alarms

image

this table could use a windoor

**Comment by MarkusLarsson421** [comment]
> you gotta decide if you want your maintenance doors to be part of their rooms or maintenance!!!

I think @PestoVerde322 made a decision regarding this. What it was I don't remember. Until then or if they don't respond it is probably better to stick to the current maps' standard by looking at other similar areas on Box Station.

File: _maps/map_files/BoxStation/BoxStation.dmm
Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

Add the doors back

Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

I wouldn't want to sit on this toilet

Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

Can you double check disposals still functions? Not sure if this is a diff rendering error

Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

Floating light bulb

Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

Ngl I really dislike these open style medbay layouts where everything is just in 1 room. It feels chaotic, unorganised and is really annoying to have structure in medbay. This is an improvement over the current box medbay where everything was dumped into a single room with no regard to making it feel like an actual medical wing, but more needs to be done.

- There shouldn't be trees/wildlife in the middle of a sterile medical bay, especially a giant tree overhanging the operating areas.
- You should spread out the stasis beds and sleepers, everything in 1 place is a nightmare to work around.
- Anywhere where people will be operated on should use white, sterile tiles and not black tiles.

I would also make the following suggestion:
- Move the chemistry zone in a bit, shrink the storage room slightly, have a room for the stasis beds with tables surrounding them.

Comment (PowerfulBacon) [inline _maps/map_files/BoxStation/BoxStation.dmm:1]: image

I know these were here already but remove these display cases, they serve no purpose for chemistry

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Overall this is a big improvement, however I think the core medbay treatment area needs some slight changes to bring it away from the medical wings that are too centralised such as RadStation where its a pain to work since you are constantly being pushed due to the tight spaces between the placement of operating beds and sleepers.

We should aim for medicals which while compact, don't have crowding issues and aren't giant, empty open areas. My least favourite medbays are RadStation and the current Box since they have those issues.

File: _maps/map_files/BoxStation/BoxStation.dmm
Comment (PestoVerde322) [reply _maps/map_files/BoxStation/BoxStation.dmm:1]: To be fair, i prefeer an open style bar like this, altought it's its own location it also somewhat feels more like a proper cafeteria if anything (considering also the "separation" doors have been pushed back between kitchen and botany.

Comment (ClownMoff) [reply _maps/map_files/BoxStation/BoxStation.dmm:1]: That last thing would mean redoing everything from scratch :S

**Comment by ClownMoff** [comment]
This shit got broken to a point that I can't repair, time to create another branch and PR

**Pull request closed without merging**